nat-lab/tests/utils/netcat.py

```diff
@@ -0,0 +1,220 @@
+import asyncio
+from config import LIBTELIO_BINARY_PATH_MAC_VM, LIBTELIO_BINARY_PATH_WINDOWS_VM
+from contextlib import asynccontextmanager
+from datetime import datetime
+from typing import Optional, AsyncIterator
+from utils.connection import Connection, TargetOS
+from utils.output_notifier import OutputNotifier
+from utils.process import Process
+from utils.python import get_python_binary
+
+
+def _get_netcat_script_path(connection: Connection) -> str:
+    if connection.target_os == TargetOS.Windows:
+        return LIBTELIO_BINARY_PATH_WINDOWS_VM + "netcat.py"
```

Comment:
Please either copy it to the windows vm or raise NotImplementedError here

Reply:
Yup, I think this was leftover from when I was testing it on windows 😞